### PR TITLE
Merge a sequence of three `test`s into a single `multi_test`

### PR DESCRIPTION
**Pull request by leonerd**
Avoids inter-test related state due to shared fixtures. Makes it easier to extend the invite test in PR #65 later.

### CODE REVIEW

File: tests/30rooms/06invite.pl [hunk 1]
```diff
@@ -41,65 +41,42 @@ sub inviteonly_room_fixture
...
-         ->main::expect_http_403;
-   };
-
-my $invited_user_fixture = local_user_fixture();
-
-test "Can invite users to invite-only rooms",
-   requires => [ $creator_fixture, $invited_user_fixture, $inviteonly_room_fixture,
-                qw( can_invite_room )],
+multi_test "Can invite users to invite-only rooms",
+   requires => [ $creator_fixture, local_user_fixture(),
```
Comment (illicitonion) [inline tests/30rooms/06invite.pl]: Any reason not to use an independent creator too? (And in the other test below)

Comment (leonerd) [inline tests/30rooms/06invite.pl]: Ah; fair point. Done at a7ec32c. PTAL.

**Comment by illicitonion** [comment]
LGTM

**Comment by leonerd** [comment]
Jenkins appears unhappy but that's an unrelated test breakage - I'll merge this one in now.

**Pull request merged**